### PR TITLE
73 adjust client carousel component style

### LINKED ISSUE #73: adjust-client-carousel-component-style
adjust-client-carousel-component-style from develop branch, since the last issue related to this component was merge to that branch.

### PR DESCRIPTION
**Pull request by Roxha-V**
client-carousel-component style adjuted to general desing

### CODE REVIEW

File: app/components/clients_carousel/index.jsx [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
               Confian en Nosotros
             </h1>
           </div>
-          <div>
+          <div class="relative origin-center mySwiper">
             <Swiper
+              freeMode={true}
               loop={true}
-              speed={2000} // ( ej 2000 ms = 2 segundos)
+              speed={3000}
```
Comment (devPalter) [inline app/components/clients_carousel/index.jsx]: change to 1600

File: app/components/clients_carousel/index.jsx [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
               autoplay={{
-                delay: 3000, // tiempo entre transiciones
```
Comment (devPalta) [inline app/components/clients_carousel/index.jsx:22]: change to 600

File: app/components/clients_carousel/index.jsx [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
-                disableOnInteraction: true, //inact en interaccion
-              }}
-              spaceBetween={0}
-              //centeredSlides={true}
-              pagination={{
-                type: "bullets",
-                dynamicBullets: "true",
-                dynamicMainBullets: "3",
+                delay: 0,
```
Comment (devPalter) [inline app/components/clients_carousel/index.jsx]: add delay at 600

File: app/components/clients_carousel/index.jsx [hunk 4]
```diff
@@ -1,35 +1,31 @@
...
+                disableOnInteraction: true,
```
Comment (devPalter) [inline app/components/clients_carousel/index.jsx]: we should add 
pauseOnMouseEnter | boolean | false | When enabled autoplay will be paused on pointer (mouse) enter over Swiper container.
-- | -- | -- | --


[pauseOnMouseEnter](https://swiperjs.com/swiper-api#param-pauseOnMouseEnter)	boolean	false	
When enabled autoplay will be paused on pointer (mouse) enter over Swiper container.

File: app/components/clients_carousel/index.jsx [hunk 5]
```diff
@@ -1,35 +1,31 @@
...
 function Clients_Carousel({ data }) {
   return (
     <section id="clients">
-      <div className="w-full h-full">
-        <div className="w-full  bg-gray-100">
+      <div className="w-full h-full bg-white drop-shadow-md pt-8 pl-8 rounded-md overflow-hidden">
+        <div className="w-full ">
           <div>
-            <h1 className="relative p-8 text-3xl font-extrabold text-black text-center">
+            <h1 className="relative p-8 mb-2 leading-8 text-4xl font-extrabold  text-center tracking-tight underline underline-offset-8 decoration-4 decoration-indigo-600">
```
Comment (devPalter) [inline app/components/clients_carousel/index.jsx]: decoration and offset not work together

**Comment by devPalter** [comment]
marqusina effect is not a good practice of user experience to capture the user's attention, sometimes constantly moving content can be annoying and make users feel rushed.

**Comment by Roxha-V** [comment]
The requested changes were made, in addition to a new proposal for the h1 title of the component. 
I understand the comment about ui and I understand that it may not be the best option. I used it considering that by having many items inside,  it seemed that more static, and only  moved by the user could be boring, or that there would not be as much attention to the component. Also i considered that I wanted to give a feeling of continuity in the movement of the carousel, where the user, if they wanted to interact, could directly touch the carousel and have it adapt to that. We can rediscuss how it could look if we need to change it.

**Review by devPalter — APPROVED** [review]

**Pull request merged**